app/models/form/sales/questions/staircase.rb

```diff
@@ -2,9 +2,9 @@ class Form::Sales::Questions::Staircase < ::Form::Question
   def initialize(id, hsh, page)
     super
     @id = "staircase"
-    @copy_key = "sales.sale_information.staircasing"
+    @copy_key = form.start_year_2025_or_later? ? "sales.setup.staircasing" : "sales.sale_information.staircasing"
     @type = "radio"
-    @answer_options = ANSWER_OPTIONS
+    @answer_options = form.start_year_2025_or_later? ? ANSWER_OPTIONS.except("3") : ANSWER_OPTIONS
```

Comment:
I am wondering if writing this differently would make it easier to tell at a glance which option was no longer available (rather than needing to work out which one 3 is) - I'd be tempted to pull it out into a function and spell things out as we do elsewhere. Not sure how worth it it is given this is relatively simple though.

Reply:
Ye that makes sense, put it in an answer options function similar to elsewhere